remote-messaging/remote-messaging-impl/src/main/java/com/duckduckgo/remote/messaging/impl/newtab/RemoteMessageViewModel.kt

```diff
@@ -172,7 +171,7 @@ class RemoteMessageViewModel @Inject constructor(
             is Dismiss -> Command.DismissMessage
             is PlayStore -> Command.LaunchPlayStore(this.value)
             is Url -> Command.SubmitUrl(this.value)
-            is UrlInContext -> Command.SubmitUrlInContext(this.value)
+            is UrlInContext -> Command.SubmitUrl(this.value)
```

Comment:
`UrlInContext` is handled in the same way as `Url` for remote messages that are not shown on MODAL surface.